libmscore/lyrics.cpp

```diff
@@ -737,8 +737,8 @@ void LyricsLineSegment::layout()
       if (isEndMelisma) {                 // melisma
             _numOfDashes = 1;
             rypos()  -= lyricsLine()->lineWidth().val() * sp * HALF; // let the line 'sit on' the base line
-            // reduce length to have some 'air' after the line
-            rxpos2() -= score()->styleD(StyleIdx::minNoteDistance) * sp;
+            // extend slightly after the chord
+            rxpos2() += score()->styleD(StyleIdx::minNoteDistance) * sp;
```

Comment:
it did look OK for cross-system-break melismas. With that change normal melismas are still too short in some cases.
![melisma](https://cloud.githubusercontent.com/assets/1786669/5717440/2abf31d0-9b01-11e4-9039-f8eea2061452.png)

Travis was in a bad mood again though...